notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -83,7 +83,9 @@
     "import hera_pspec as hp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    USE_CORR_MATRIX = True
</pre><p>This should be an environment variable and should also control whether your whole big calculation gets run... assuming we want it to be an option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='104'/>

Reply:
Done